vagrant/Vagrantfile

```diff
@@ -44,13 +45,16 @@ Vagrant.configure(2) do |config|
     aws.session_token = ENV['AWS_SESSION_TOKEN']
     aws.associate_public_ip = true
     aws.tags = { 'Name' => (ENV['DEPLOY_ENV'] || ENV['USER']) + " concourse" }
+    aws.keypair_name = ENV['VAGRANT_SSH_KEY_NAME']
```

Comment:
It's required as, we have deprecated a Vagrant box image we've been using thus far, which has been a resource of an unknown origin and we've been worried it will disappear anytime. Other side of the medal, is the fact that the box had "hardcoded" ssh key, which could mean, for the time the box is up anyone with that key would have an access to the VM in the time of its live.

With our new approach, we're creating the key on the go with AWS Key Pairs and then removing it in the destroy action. We're also able, to use official box (Ubuntu) as oppose to rely on third-party images.